screens/HoursScreen.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import {
+  ScrollView,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+import * as firebase from 'firebase';
+
+import Days from '../constants/Days';
+
+export default class HoursScreen extends React.Component {
+  static navigationOptions = {
+    header: null
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      daysToHours: {},
+    };
+  }
+
+  async componentDidMount() {
+    var daysToHours = {};
+    await firebase
+      .database()
+      .ref('info')
+      .once('value')
+      .then(function(data) {
+        const DB_DAYS_KEYS_TO_READABLE_DAYS = Days.DB_DAYS_KEYS_TO_READABLE_DAYS;
+        data.forEach(function(childNode) {
+          var dayKey = childNode.key;
+          if (DB_DAYS_KEYS_TO_READABLE_DAYS.hasOwnProperty(dayKey)) {
+            var day = childNode.val();
+            var readableDay = DB_DAYS_KEYS_TO_READABLE_DAYS[dayKey];
+            daysToHours[readableDay] = {
+              day: readableDay,
+              hours: day['12hours'],
+              restock: day['restock'],
+            };
+          }
+        });
+        this.setState({
+          daysToHours: daysToHours
+        });
+      }.bind(this));
+  }
+
+  getEmojis(restock) {
+    let arr = new Array();
+    Object.keys(EMOJIS).map(function(key, emoji){
```

Comment:
Also, looks like "emoji" variable isn't used. You can just have function(key) and that will be fine.